src/app/profile/page.tsx

```diff
@@ -0,0 +1,23 @@
+import FileInput from '@/components/pages/profile/file-input'
```

Comment:
Fig design to match Figma

![CopyQ rBitVF](https://github.com/dev-KPI/messenger-frontend/assets/95881717/778ef580-1ac5-4898-8b35-a39132f6cf24)

![CopyQ zhFRgC](https://github.com/dev-KPI/messenger-frontend/assets/95881717/2899a811-c65e-42d9-8efe-2a615fe3099c)

Reply:
Wow, Anton can confirm that I fixed this issue yesterday... Nevermind, thank's for your attention, I'll fix this.